aspnetcore/security/authentication/azure-ad-b2c.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Cloud authentication with Azure Active Directory B2C
+author: camsoper
+description: Shows how to set up Azure Active Directory B2C with ASP.NET Core
+keywords: ASP.NET Core,authentication,AAD,B2C,Azure Active Directory,Active Directory
+ms.author: casoper
+manager: wpickett
+ms.date: 01/12/2018
+ms.topic: article
+ms.technology: aspnet
+ms.prod: asp.net-core
+uid: security/authentication/azure-ad-b2c
+---
+# Cloud authentication with Azure Active Directory B2C
+
+By [Cam Soper](https://twitter.com/camsoper)
```

Comment:
do we add bylines to articles that we (FTEs) write? I thought we only used those when we converted external articles to a doc or have multiple authors.

Reply:
Over here, it's been done since the beginning. I guess for consistency with community-written topics?

As a side note: Glad they didn't remove the bylines (at least for community authors). *Greatly appreciated!*

Reply:
Maybe, not sure @guardrex. Just keep it then @CamSoper. It was just my curiosity.